quick-start/docker-compose.yml

```
@@ -0,0 +1,65 @@
+version: '2'
+networks:
+  bridge:
+    driver: bridge
+services:
+  web:
+    # Use an already built XWiki image from DockerHub.
+    image: "xwiki:13.3-postgres-tomcat" 
+    container_name: xwiki-postgres-tomcat-web
+    depends_on:
+      - db
+    ports:
+      - "8080:8080"
+      - "8443:8443"  
+    # The DB_USER/DB_PASSWORD/DB_HOST variables are used in the hibernate.cfg.xml file.
+    environment:
+      - DB_USER=xwiki
+      - DB_PASSWORD=xwiki
+      - DB_DATABASE=xwiki
+      - DB_HOST=xwiki-postgres-db
```

This is a question of style, but I personally prefer the key/value approach to the list. Each element on this list is a String which docker splits on the first `=` it finds. While this works exactly like it would for shell envs, I find it less intuitive in a YAML file. The key value approach would look like this:
```suggestion
      DB_USER: xwiki
      DB_PASSWORD: xwiki
      DB_DATABASE: xwiki
      DB_HOST: xwiki-postgres-db
```

I'll leave it up to you, just keep it consistent across the whole file.